-------------------------------------------------------------------------------
Test set: com.examly.springapp.SpringApplicationTests
-------------------------------------------------------------------------------
Tests run: 4, Failures: 2, Errors: 2, Skipped: 0, Time elapsed: 6.826 s <<< FAILURE! - in com.examly.springapp.SpringApplicationTests
test_case1  Time elapsed: 0.408 s  <<< ERROR!
org.springframework.web.util.NestedServletException: Request processing failed; nested exception is org.springframework.orm.jpa.JpaSystemException: ids for this class must be manually assigned before calling save(): com.examly.springapp.model.Task; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.examly.springapp.model.Task
	at com.examly.springapp.SpringApplicationTests.test_case1(SpringApplicationTests.java:30)
Caused by: org.springframework.orm.jpa.JpaSystemException: ids for this class must be manually assigned before calling save(): com.examly.springapp.model.Task; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.examly.springapp.model.Task
	at com.examly.springapp.SpringApplicationTests.test_case1(SpringApplicationTests.java:30)
Caused by: org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.examly.springapp.model.Task
	at com.examly.springapp.SpringApplicationTests.test_case1(SpringApplicationTests.java:30)

test_case2  Time elapsed: 0.285 s  <<< FAILURE!
java.lang.AssertionError: No value at JSON path "$[*].houseNo"
	at com.examly.springapp.SpringApplicationTests.test_case2(SpringApplicationTests.java:48)

test_case3  Time elapsed: 0.02 s  <<< ERROR!
org.springframework.web.util.NestedServletException: Request processing failed; nested exception is java.util.NoSuchElementException: No value present
	at com.examly.springapp.SpringApplicationTests.test_case3(SpringApplicationTests.java:58)
Caused by: java.util.NoSuchElementException: No value present
	at com.examly.springapp.SpringApplicationTests.test_case3(SpringApplicationTests.java:58)

test_case4  Time elapsed: 0.011 s  <<< FAILURE!
java.lang.AssertionError: Status expected:<200> but was:<405>
	at com.examly.springapp.SpringApplicationTests.test_case4(SpringApplicationTests.java:79)

